backend/api/pretix/mutations.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+import strawberry
+from strawberry.types import Info
+
+import pretix
+from api.pretix.permissions import IsTicketOwner
+from api.pretix.query import get_user_tickets
+from api.pretix.types import AttendeeTicket, UpdateAttendeeTicketInput
+from conferences.models.conference import Conference
+
+logger = logging.getLogger(__name__)
+
+
+@strawberry.type
+class TicketReassigned:
+    id: strawberry.ID
+    email: str
+
+
+UpdateAttendeeTicketResult = strawberry.union(
+    "UpdateAttendeeTicketResult",
+    (TicketReassigned, AttendeeTicket),
+)
+
+
+@strawberry.type
+class AttendeeTicketMutation:
+    @strawberry.mutation(permission_classes=[IsTicketOwner])
+    def update_attendee_ticket(
+        self, info: Info, conference_code: str, input: UpdateAttendeeTicketInput
```

Comment:
```suggestion
        self, info: Info, conference_code: str, input: UpdateAttendeeTicketInput
```

is this `conference_code` consistent with other APIs?

Reply:
Think only [order](https://github.com/pythonitalia/pycon/blob/main/backend/api/orders/query.py#L13) query. 
I will switch back to `conference`...